atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -398,7 +398,8 @@ def add_log_type(output):
         metadata_version = 0
 
         worker_metadatas = self.workflow.postbuild_results.get(PLUGIN_FETCH_WORKER_METADATA_KEY)
-        remote_source_file_outputs, kojifile_components = get_maven_metadata(self.workflow)
+        kojifile_components, remote_source_file_outputs, pnc_build_ids = get_maven_metadata(
```

Comment:
maybe get_maven_metadata shouldn't even return those pnc_build_ids

and it can all be done in '_update_extra' in KojiImportPlugin

Reply:
removed from get_maven_metadata and moved the whole operation to a separate function.